src/game/Globals/ObjectMgr.h

```diff
@@ -421,6 +421,11 @@ class HonorStanding
         {
             return honorPoints > rhs.honorPoints;
         }
+	
+        operator bool() const
```

Comment:
Typically a good practice is to make conversions like this ```explicit``` so you don't get conversions "by accident" to say so. They would still be "contextually converted to bool" so you could continue to use this in if, while, for, built-in logical operators, and so on.